### PR TITLE
Handle new user kyc fields (Submit Application API)

### PR DESCRIPTION
**Pull request by sagarrakshe**
Todo tasks:
* Add Deduplication policy, also take into account mutiple kyc records for the same user.
* Fix bug in fetching kyc info
* Fix bug for fields `domicile_same` and `face_similarity`
* Correct the tests

### CODE REVIEW

**Comment by nishantmodak** [comment]
@sagarrakshe no tests?

File: common.go [hunk 1]
```diff
@@ -26,6 +26,9 @@ func BasicAuth(username, password string) string {
 
 func (client *Client) MakeRequest(reqType, url string, payload interface{}, response interface{}) error {
 	b, err := json.Marshal(payload)
+	log.Println(reqType)
```
Comment (sagarrakshe) [inline common.go]: Remove if not required.

File: fineract_client.go [hunk 2]
```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	fineractHost     = "https://10.10.8.47:8443"
+	fineractHost     = "https://localhost:8443" //"https://10.10.8.47:8443"
```
Comment (sagarrakshe) [inline fineract_client.go]: localhost?

File: kycinfo.go [hunk 3]
```diff
@@ -153,14 +178,20 @@ func (client *Client) GetKycInfoByID(r *GetKycInfoByIDRequest) (*GetKycInfoByIDR
 
 // Create KYC info
 func (client *Client) CreateKYCInfo(r *KycInfoCreateRequest) (*CreateKycInfoResponse, error) {
+	if r.ExtraInfos == "" {
+		r.ExtraInfos = "none"
```
Comment (sagarrakshe) [inline kycinfo.go]: Extrainfos in fineract can be nullable, so no need to handle here.

File: kycinfo.go [hunk 4]
```diff
@@ -169,14 +200,20 @@ func (client *Client) CreateKYCInfo(r *KycInfoCreateRequest) (*CreateKycInfoResp
 
 // Update KYC info
 func (client *Client) UpdateKYCInfo(r *KycInfoUpdateRequest) (*CreateKycInfoResponse, error) {
+	if r.ExtraInfos == "" {
+		r.ExtraInfos = "none"
```
Comment (sagarrakshe) [inline kycinfo.go]: Same here.

File: kycinfo.go [hunk 5]
```diff
@@ -196,15 +233,53 @@ func (client *Client) rowToKYC(row []interface{}) (*KycInfo, error) {
 	if nil != err {
 		return nil, err
 	}
+
+	income := int64(0)
+	faceSimilarity := 0.0
+
```
Comment (sagarrakshe) [inline kycinfo.go]: `var income int64` could be the better way to intialize

File: kycinfo.go (hunk 3 above)
Comment (sapanakothari) [reply kycinfo.go]: `extra_infos` text NOT NULL,

not-nullable at the moment - fixing it

File: kycinfo.go (hunk 4 above)
Comment (sapanakothari) [reply kycinfo.go]: same as above

**Pull request merged**